docs/building-experiments/experiments/creating-experiments.md

```diff
@@ -14,38 +14,44 @@ Experiments are a set of metrics that correspond to users being shown different
 
 ![Fill experiment form](../../../static/img/building-experiments/fill-create-experiment-form.png)
 
-Give your experiment a name, start and end date. You can also choose to set an event analysis period that is different from your assignment period. This is useful if you want to analyze the impact of an experiment on long-term metrics.
+Give your experiment a name and hypothesis. 
 
-3. Navigate to the **Set Up** tab and click the **Configure the Experiment** button
+3. Click the **Configure the Experiment** button
 
 ![Configure experiment](../../../static/img/building-experiments/set-up-and-configure-experiment.png)
 
-6. Select an assignment SQL from the definitions you created
+4. Select the date range for your experiment.
+
+![Select dates](../../../static/img/building-experiments/select-dates.png)
+
+You can also choose to set a metric event analysis period that is different from your assignment period. This is useful if you want to analyze the impact of an experiment on long-term metrics.
```

Comment:
Do you want to link to the reference document here? I feel like it would be hard to find otherwise